### PR TITLE
Issue/11153 (multiple images) update preview imageview + thumbnail after crop

### LINKED ISSUE #11153: Add support for editing multiple local images from the Hub before they are uploaded
A horizontal RecyclerView on the Hub is added in https://github.com/wordpress-mobile/WordPress-Android/issues/11152. Goal of this task is to add support for editing the images shown in the RecyclerView and appending them to Gutenberg. 

- [x] Make sure clicking on Edit opens the selected image
- [x] Make sure the ImageView and thumbnails gets udpated after the user edits the image
- [x] Make sure all images are added to the editor when the user clicks on "Insert"

This is an overall design of the flow (Android specific design will be added later):
Screenshot 2020-01-23 at 19 04 39

~Note: Gutenberg currently doesn't support selecting multiple images from the local storage. The flow added in this ticket might not be reachable by the user until the support is added to Gutenberg.~
This is supported now in Gallery block.

### PR DESCRIPTION
**Pull request by ashiagr**
Partially fixes #11153

To update preview image view and thumbnails after the user edits the image.

PR submission checklist:

- [X] I have considered adding unit tests where possible.
- [X] I have considered adding accessibility improvements for my changes.
- [X] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/crop/CropFragment.kt [hunk 1]
```diff
@@ -57,11 +59,15 @@ class CropFragment : Fragment(), UCropFragmentCallback {
     private fun initializeViewModels() {
         viewModel = ViewModelProvider(this).get(CropViewModel::class.java)
         setupObservers()
-        viewModel.start(navArgs.inputFilePath, navArgs.outputFileExtension, requireContext().cacheDir)
+        viewModel.start(
+            navArgs.inputFilePath,
+            navArgs.outputFileExtension,
+            File(requireContext().cacheDir, "media_editing")
```
Comment (malinajirka) [inline libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/crop/CropFragment.kt:65]: We might consider passing just the `requireContext().cacheDir` as it requires Context and creating the subfolder within the VM, wdyt? We might also consider creating a constant for the `media_editing`. Wdyt?

File: libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/preview/PreviewImageFragment.kt [hunk 2]
```diff
@@ -194,27 +202,40 @@ class PreviewImageFragment : Fragment() {
...
+        viewModel.navigateToCropScreenWithFileInfo.observe(viewLifecycleOwner, Observer { fileInfoEvent ->
             fileInfoEvent?.getContentIfNotHandled()?.let { fileInfo ->
                 navigateToCropScreenWithFileInfo(fileInfo)
             }
         })
+
+        findNavController().currentBackStackEntry?.savedStateHandle
+                ?.getLiveData<CropResult>(CropFragment.CROP_RESULT)?.observe(
+                viewLifecycleOwner, Observer { result ->
+            Log.d(PreviewImageFragment::javaClass.name, "result")
```
Comment (malinajirka) [inline libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/preview/PreviewImageFragment.kt:228]: This log slipped through during the pair programming session. I assume we'll want to remove it.

**Review by malinajirka — APPROVED** [review]
Thanks @ashiagr!

I've left minor comments but I'm approving this PR. Feel free to merge it either without making any changes or after you address the comments - I don't expect it'll need additional review (we are merging it into a feature branch anyway).

File: libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/preview/PreviewImageFragment.kt (hunk 2 above)
Comment (ashiagr) [reply libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/preview/PreviewImageFragment.kt]: Removed here: 557bb3e

File: libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/crop/CropFragment.kt (hunk 1 above)
Comment (ashiagr) [reply libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/crop/CropFragment.kt]: Done here 5d57f28

**Pull request merged**